### PR TITLE
[13.0][IMP] mrp_multi_level_estimate: Estimates and other demand sources strategy

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Introduces a new concept to define the strategy to apply when estimates coexist with other sources of demand.

Also, remove unneeded dependency to date_range and relicense to LGPL.

@ForgeFlow

### CODE REVIEW

File: mrp_multi_level_estimate/models/mrp_area.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        default="all",
+        help="Define the strategy to follow in MRP multi level when there is a"
+        "coexistence of demand from demand estimates and other sources.\n"
+        "* Always consider all sources: nothing is excluded or ignored.\n"
+        "* Ignore other sources for products with estimates: When there "
+        "are estimates entered for product and they are in a present or "
+        "future period, all other sources of demand are ignored for those "
+        "products.\n"
+        "* Ignore other sources during periods with estimates: When "
+        "you create demand estimates for a period and product"
```
Comment (BernatPForgeFlow) [inline mrp_multi_level_estimate/models/mrp_area.py]: Small comment to improve the readability of the helper, add the comma and the space between 'product' and 'other'.

**Review by BernatPForgeFlow — APPROVED** [review]
Functional test OK!

File: mrp_multi_level_estimate/models/mrp_area.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply mrp_multi_level_estimate/models/mrp_area.py]: Done, thanks!

**Comment by LoisRForgeFlow** [comment]
Aborted merge to use minor bump instead.

/ocabot merge minor

**Pull request merged**